### ISSUE TITLE
Pytorch library error

### ISSUE THREAD

**Issue by Lorena-J** [MouseLand/Kilosort#653]
### Describe the issue:

Hi! I'm triyng to start the sorting with kilosort4 but there is an error with the pytorch library, i try to re-install kilosrt from fresh and the error continues, i try to convert a file and it works, but you cannot run the kilosort to do the spikesorting process. I have cuda 11.8 version and pytorch 2.2.2 version, i try also with the pytorch 2.2.1 and the cuda 12 but it doesent fix the problem, and uninstall pytroch and install it again doesen't work either.
<img width="828" alt="Captura de pantalla 2024-04-03 a las 8 36 29" src="https://github.com/MouseLand/Kilosort/assets/146965710/77ff47d2-29ef-400b-9c38-adb988f26cc4">
(The error in the miniconda prompt)
<img width="456" alt="Captura de pantalla 2024-04-03 a las 8 41 16" src="https://github.com/MouseLand/Kilosort/assets/146965710/d3024e15-21a1-49e1-b19b-511b5f5acab8">
(The error in kilosort prompt)

**Comment by jacobpennington** [comment]
Try clicking on "reset GUI" and "clear cache," then close the GUI and open it again and then select the converted file. I've noticed this happens sometimes after using the file conversion tool.

**Comment by Lorena-J** [comment]
@jacobpennington I try it and it doesn't work, I get the same error.

**Comment by jacobpennington** [comment]
@Lorena-J In that case, are you able to share a sample of data that I could use to debug this? Preferably in the original file format so that I can check the conversion process as well.

**Comment by Lorena-J** [comment]
Of course!! Here you have a sample in the original format file, it comes from Open Ephys:
[sample of data](https://drive.google.com/drive/folders/1Q0dWhvKL5Bz-VnSqFydeV5vzMlqOLPUv?usp=drive_link)

**Comment by jacobpennington** [comment]
I was able to convert the data and load it fine with a stand-in 64-channel probe file. However, I did reproduce the error when I tried to load it using the default 385-channel neuropixels 1 probe, which could have happened if you have "Auto Load" checked and re-loaded the GUI after converting. Can you try unchecking "Auto Load," restarting the GUI, and then choose your data and probe again? I will also update the GUI code to prevent that from happening.

**Comment by Lorena-J** [comment]
I get the same error unchecking the 'auto load', I noticed that when I press the 'LOAD' button the error appears, but if I don't press the 'LOAD' button and I press the 'run' button it doesn't start doing anything.

**Comment by jacobpennington** [comment]
Can you please post a screenshot of what the entire GUI looks like just before you click on 'LOAD'?

**Comment by Lorena-J** [comment]
Yes, of course, here you have a screenshot before click 'LOAD', in that moment I can press 'rub' button but it doesn't works, when I click 'run' Idon't get any error, just it doesn't work. 
<img width="1354" alt="Captura de pantalla 2024-04-12 a las 9 25 54" src="https://github.com/MouseLand/Kilosort/assets/146965710/410ee679-e7fc-467b-a0b5-e8802984f284">
And this other screenshot is when I click 'LOAD' and the error i get in kilosort terminal
<img width="1353" alt="Captura de pantalla 2024-04-12 a las 9 26 18" src="https://github.com/MouseLand/Kilosort/assets/146965710/0f05aaff-9d65-4b56-bf5e-cba7b8658d33">

<img width="442" alt="Captura de pantalla 2024-04-12 a las 9 29 25" src="https://github.com/MouseLand/Kilosort/assets/146965710/f462b0f8-efb1-494e-a5ec-d94004be8686">

**Comment by jacobpennington** [comment]
Okay, sorry this is being so stubborn. Are you able to send the exact .bin and .prb files you're using there so I can check it with those? I can't see any reason why this should load for me but not you.

**Comment by Lorena-J** [comment]
Sure! here you have the .prb and the .bin (the .bin file was converted with kilosort4 conversor, from open ephys .continuous files). The original .continuous are also in the folder:
[all files](https://drive.google.com/drive/folders/1Q0dWhvKL5Bz-VnSqFydeV5vzMlqOLPUv?usp=drive_link)

**Comment by jacobpennington** [comment]
Okay, I found the problem. It's not loading because your channel map is numbered 1 to 64 instead of 0 to 63, as I mentioned in one of the other issues you posted. That's causing the error, it's just not a very helpfully worded error, but I'll add something to catch that in the future.

Something I noticed while I was looking at this, which isn't related to this error but might cause other problems: it looks like one of the contacts in your probe file might be specified incorrectly. I see this in the GUI:
![image](https://github.com/MouseLand/Kilosort/assets/35777376/3cbf7969-0321-4d44-85d9-c06c67ab2638)

**Comment by jacobpennington** [comment]
I'm going to close this for now since the error has been figured out, please let us know if you have any more trouble with this.
